### ISSUE TITLE
glob pattern does not work

### ISSUE THREAD

**Issue by bitranox** [rsionnach/sloppylint#11]
neither of that works : 

```
sloppylint -f json -i ./scripts/**
sloppylint -f json -i scripts/**
sloppylint -f json -i **/scripts/**

```

**Comment by rsionnach** [comment]
Thanks for the report! The `-i` flag is for **ignore** patterns (to exclude files), not include patterns.

In **v0.4.0** (just released), we've added a new `--include/-I` option specifically for filtering to only scan certain files:

```bash
# Scan only files in scripts directory
sloppylint -I 'scripts/**/*.py' .

# Scan only test files
sloppylint -I '**/test_*.py' .
```

You can also configure this in `pyproject.toml`:

```toml
[tool.sloppy]
include = ['scripts/**/*.py']
```

Please upgrade to v0.4.0 (`pip install --upgrade sloppylint`) and try the new `-I/--include` option. Let me know if this addresses your use case!

**Comment by bitranox** [comment]
I knew that it was for exclusion - but it simply does not work ;-)
the directory does not get excluded !"

**Comment by bitranox** [comment]
(pycharm) srvadmin@lxc-pydev:/media/srv-main-softdev/projects/libs/lib_layered_config$ sloppylint -f json -i scripts/**
{
  "summary": {
    "total_issues": 34,
    "score": {
      "noise": 0,
      "quality": 16,
      "style": 208,
      "structure": 203,
      "total": 427
    },
    "verdict": "DISASTER"
  },
  "issues": [
    {
      "pattern_id": "god_function",
      "severity": "high",
      "axis": "style",
      "file": "scripts/_utils.py",
      "line": 125,
      "column": 0,
      "message": "Function has 6 parameters (max 5)",
      "code": "def run(...)"
    },
    {
      "pattern_id": "god_function",
      "severity": "high",
      "axis": "style",
      "file": "scripts/_utils.py",
      "line": 403,
      "column": 0,
      "message": "Function has 59 lines (max 50)",
      "code": "def get_project_metadata(...)"
    },
    {
      "pattern_id": "god_function",
      "severity": "high",
      "axis": "style",
      "file": "scripts/_utils.py",
      "line": 504,
      "column": 0,
      "message": "Function has 78 lines (max 50)",
      "code": "def _render_metadata_module(...)"
    },
    {
      "pattern_id": "broad_except",
      "severity": "high",
      "axis": "structure",
      "file": "scripts/_utils.py",
      "line": 750,
      "column": 4,
      "message": "Broad exception handling - catch specific exceptions instead",
      "code": "except Exception:"

**Comment by bitranox** [comment]
and i would propose -x/--exclude and -i/--include
